Tribler/Core/CacheDB/SqliteCacheDBHandler.py

```diff
@@ -1525,6 +1525,17 @@ def on_remove_torrent_from_dispersy(self, channel_id, dispersy_id, redo):
 
         self.notifier.notify(NTFY_CHANNELCAST, NTFY_UPDATE, channel_id)
 
+        sql = """SELECT infohash, dispersy_cid FROM Torrent, _ChannelTorrents, Channels
+        WHERE Torrent.torrent_id = _ChannelTorrents.torrent_id
+        AND _ChannelTorrents.channel_id = ? AND _ChannelTorrents.dispersy_id = ?
+        AND Channels.id = _ChannelTorrents.channel_id"""
+        infohash, dispersy_cid = self._db.fetchone(sql, (channel_id, dispersy_id))
+
+        if infohash:
+            self.notifier.notify(NTFY_TORRENTS, NTFY_DELETE, None,
```

Comment:
I want to keep consistency with the torrent discovered events since that's closely related to this new event.

Also, I think this third argument is a bit weird and I argue it's easier if that argument is removed. Moreover, it is often not used. Our notification system is not designed around these use-cases (I believe it was originally designed to work with database events, hence the update/insert operations). Now, it leads to confusion.